src/main/webapp/index.html

```diff
@@ -28,7 +28,8 @@
         <li><a href="/">Home</a></li>
       </ul>
     </nav>
-    <h1> This is Stanley's first change</h1>
+    <h1> This is Stanley's first change! Everthing here is pretty cool</h1>
+    <h2> Team 10 is the best Team ever </h2>
```

Comment:
I like this new subheading a lot, but it's not related to the messages changes. Could you undo the changes to index.html in this PR, and make a new branch with a different PR if you'd like to change this file?

Reply:
Sure I'm on it right now